<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Maintenance</class>
 <widget class="QDialog" name="Maintenance">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1292</width>
    <height>636</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
font-family:century gothic;
font-size :20px;

}
 QTabWidget{
background:#333;
border-radius:15px;
}

QLabel {
postion:center;
color :#CD5C5C;
}
QTableView {
background:white;
border-radius:15px;
font-size :20px;

}
QPushButton {
background: #CD5C5C;border:none;
color:white;
 border-radius: 15px;
}
QPushButton:hover
{ color:white;
 border-radius: 15px;
background-color :#F08080;border:none;
}</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>70</y>
     <width>1181</width>
     <height>571</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>liste des réclamations</string>
    </attribute>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>285</x>
       <y>50</y>
       <width>561</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string> Bienvenue à notre service des matérieux en panne</string>
     </property>
    </widget>
    <widget class="QTableView" name="tableView">
     <property name="geometry">
      <rect>
       <x>35</x>
       <y>230</y>
       <width>871</width>
       <height>192</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="filter_line_2">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>150</y>
       <width>321</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
background:transparent;
border:none;
color: #717072;
border-bottom:1px solid #717072;
}</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>140</y>
       <width>341</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Rechercher un matériel en panne</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>820</x>
       <y>140</y>
       <width>241</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Avec</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>900</x>
       <y>160</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Matériel en panne</string>
    </attribute>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>10</y>
       <width>531</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Envoyer un matériel en panne au maintenance</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="etat">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>200</y>
       <width>211</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
background:transparent;
border:none;
color: #717072;
border-bottom:1px solid #717072;
}</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>140</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Envoyer</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="code">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>100</y>
       <width>211</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
background:transparent;
border:none;
color: #717072;
border-bottom:1px solid #717072;
}</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>211</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Code materiel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>210</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Etat materiel</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="probleme">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>150</y>
       <width>211</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
background:transparent;
border:none;
color: #717072;
border-bottom:1px solid #717072;
}</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Probléme </string>
     </property>
    </widget>
    <widget class="QTableView" name="tableView_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>320</y>
       <width>1101</width>
       <height>151</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">/*QTableView::item
{
anchors.fill: racesWindow
        width: 840
        height: 400
       
}
*/</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Matériel en bon etat</string>
    </attribute>
    <widget class="QComboBox" name="comboBox_2">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>210</y>
       <width>401</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="Submit_2">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>360</y>
       <width>221</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
background-color :#CD5C5C;border:none;
color:white;
border-radius:15px;
}</string>
     </property>
     <property name="text">
      <string>Libérer</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>70</y>
       <width>431</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Matériel en Bon etat</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>190</y>
       <width>271</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Entrer le code à barre</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="Submit_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color :#CD5C5C;border:none;
color:white;
border-radius:15px;
}</string>
   </property>
   <property name="text">
    <string>Retour</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
